### ISSUE TITLE
The validation loss seems too high

### ISSUE THREAD

**Issue by haoheliu** [YuanGongND/ast#75]
Thanks for making such a wonderful repo. 

When I run the model, the validation loss is very high(around 0.6940), but the mAP is keep increasing normally. Could you please explain a little bit about this?

Many thanks,

**Comment by haoheliu** [comment]
The same phenomenon in your log file: https://github.com/YuanGongND/ast/blob/master/egs/audioset/exp/test-full-f10-t10-pTrue-b12-lr1e-5/log_2090852.txt

**Comment by YuanGongND** [comment]
Hi Haohe,

Thanks for pointing this out.

I somehow know both the training and validation loss have some problems in the code. For the validation loss, one obvious reason is that we applied a Sigmoid before calculating the loss https://github.com/YuanGongND/ast/blob/d7d8b4b8e06cdaeb6c843cdb38794c1c7692234c/src/traintest.py#L280 while the `nn.BCEWithLogitsLoss()` expects raw logits. I cannot recall why I added the Sigmoid, but it shouldn't impact the mAP. 

-Yuan
